gcloud/language/client.py

```diff
@@ -54,7 +54,7 @@ def document_from_text(self, content, **kwargs):
 
         :rtype: :class:`Document`
         :returns: A plain-text document bound to this client.
-        :raises: :class:`TypeError` if ``doc_type`` is passed as a
+        :raises: :class:`~exceptions.TypeError` if ``doc_type`` is passed as a
```

Comment:
Have you tried it with the `:exc:` role?  I think it works as desired, without the `exceptions.` prefix.